frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx

```diff
@@ -0,0 +1,135 @@
+};
+const labelPlural = 'Virtual Machine Templates';
+
+const tableColumnClass = classNames('col-lg-2', 'col-sm-4', 'col-xs-4');
+const tableColumnClassHiddenOnSmall = classNames('col-lg-2', 'hidden-sm', 'hidden-xs');
+
+const tableColumnClasses = [
+  tableColumnClass,
+  tableColumnClass,
+  tableColumnClass,
+  tableColumnClassHiddenOnSmall,
+  tableColumnClassHiddenOnSmall,
+  tableColumnClassHiddenOnSmall,
+  Kebab.columnClass,
+];
+
+const VmTempleateTableHeader = () => {
+  return [
+    {
+      title: 'Name',
+      sortField: 'metadata.name',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Namespace',
+      sortField: 'metadata.namespace',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'Description',
+      sortField: 'metadata.annotations.description',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[2] },
+    },
+    {
+      title: 'Source',
+      props: { className: tableColumnClasses[3] },
+    },
+    {
+      title: 'OS',
+      props: { className: tableColumnClasses[4] },
+    },
+    {
+      title: 'Flavor',
+      props: { className: tableColumnClasses[5] },
+    },
+    {
+      title: '',
+      props: { className: tableColumnClasses[6] },
+    },
+  ];
+};
+VmTempleateTableHeader.displayName = 'VmTemplateTableHeader';
+
+const VmTemplateTableRow = ({ obj: template, index, key, style }) => {
+  const os = getTemplateOperatingSystems([template])[0];
+
+  return (
+    <TableRow id={template.metadata.uid} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>
+        <ResourceLink
+          kind={kind}
+          name={getName(template)}
+          namespace={getNamespace(template)}
+          title={template.metadata.uid}
+        />
+      </TableData>
+      <TableData className={tableColumnClasses[1]}>
+        <ResourceLink
+          kind="Namespace"
+          name={getNamespace(template)}
+          title={getNamespace(template)}
+        />
+      </TableData>
+      <TableData className={tableColumnClasses[2]}>
+        {_.get(template.metadata, 'annotations.description', DASH)}
+      </TableData>
+      <TableData className={tableColumnClasses[3]}>
+        <TemplateSource template={template} />
+      </TableData>
+      <TableData className={tableColumnClasses[4]}>{os ? os.name || os.id : DASH}</TableData>
+      <TableData className={tableColumnClasses[5]}>{getTemplateFlavors([template])[0]}</TableData>
+      <TableData className={tableColumnClasses[6]}>
+        <ResourceKebab actions={menuActions} kind={kind} resource={template} />
+      </TableData>
+    </TableRow>
+  );
+};
+VmTemplateTableRow.displayName = 'VmTemplateTableRow';
+
+const VirtualMachineTemplates = (props) => (
+  <Table
+    {...props}
+    aria-label={labelPlural}
+    Header={VmTempleateTableHeader}
+    Row={VmTemplateTableRow}
```

Comment:
I think we should be using `virtualize` for the main tables

Reply:
in https://github.com/openshift/console/pull/1673#discussion_r295886830 

@vojtechszocs I removed it because it's true by default ... did I miss something ?

Reply:
ah okay, you are right. I missed that

Reply:
np, thanks for looking :+1:

Reply:
@suomiy The `virtualize` prop should always default to `true` in Console table components.

In both `Table(Inner).render` and `List(Inner).render` methods, there's `virtualize = true` destructuring from `props`. In other words, all Console tables should be rendered as "virtualized" by default.

Reply:
I'm OK with having an explicit `virtualized` prop here, just tried to say is that its default value is `true` anyway.